CMakeLists.txt

```diff
@@ -51,15 +51,25 @@ if(NOT CMAKE_BUILD_TYPE)
   set(CMAKE_BUILD_TYPE "Dev" CACHE STRING "Choose the type of build." FORCE)
 endif()
 
-# If not in a git repo (e.g., a tarball) these tokens set the version string,
-# otherwise the result of `git describe` is used.
+# If not in a git repo (e.g., a tarball) these tokens define the complete
+# version string, else it is combined with the result of `git describe`.
 set(NVIM_VERSION_MAJOR 0)
 set(NVIM_VERSION_MINOR 1)
-set(NVIM_VERSION_PATCH 1)
+set(NVIM_VERSION_PATCH 2)
+set(NVIM_VERSION_PRERELEASE "-dev") # for package maintainers
 
 file(TO_CMAKE_PATH ${CMAKE_CURRENT_LIST_DIR}/.git FORCED_GIT_DIR)
 include(GetGitRevisionDescription)
-git_describe(NVIM_VERSION_MEDIUM)
+get_git_head_revision(GIT_REFSPEC NVIM_VERSION_COMMIT)
+if(NVIM_VERSION_COMMIT) # is a git repo
```

Comment:
@Pyrohh I had to restore this condition because otherwise NVIM_VERSION_MEDIUM gets a dummy value "HEAD-HASH-NOTFOUND".

re 7be12edd38b1da0a506f39651d8c6a6f9aa5626a